config/default-spinnaker-local.yml

```diff
@@ -45,6 +45,22 @@ providers:
     #   name: my-cf-username
     #   password: my-cf-password
 
+  kubernetes:
+    # If you want to deploy some services to Kubernetes
```

Comment:
I'm trying to remove `k8s` from everywhere, since there is no consistent reason to use the abbreviation. I'll update the sample commands to reflect this.